packages/pigeon/lib/dart_generator.dart

```diff
@@ -297,12 +308,13 @@ void generateTestDart(
   indent.writeln(
     'import \'${_escapeForDartSingleQuotedString(mainDartFile)}\';',
   );
-  for (Api api in root.apis) {
+  for (final Api api in root.apis) {
     if (api.location == ApiLocation.host && api.dartHostTestHandler != null) {
       final Api mockApi = Api(
-        name: api.dartHostTestHandler,
+        name: api.dartHostTestHandler ?? 'null',
```

Comment:
Can you use the bang here instead, we already established that it is not null in the conditional.  I don't want to accidentally get classes with "null" as their name sometime in the future.

Reply:
done